xbmc/network/httprequesthandler/HTTPImageHandler.cpp

```diff
@@ -37,7 +37,25 @@ CHTTPImageHandler::CHTTPImageHandler(const HTTPRequest &request)
     XFILE::CImageFile imageFile;
     const CURL pathToUrl(file);
     if (imageFile.Exists(pathToUrl))
+    {
       responseStatus = MHD_HTTP_OK;
+      struct __stat64 statBuffer;
+      if (imageFile.Stat(pathToUrl, &statBuffer) == 0)
+      {
+        struct tm *lastChange;
+#ifdef HAVE_LOCALTIME_R
```

Comment:
aha. somebody has already adressed the issue. that's the best fixes ever don't you agree? :)